heat/examples/vm_two_nics_with_share.yaml

```diff
@@ -0,0 +1,97 @@
+heat_template_version: 2018-03-02
+
+description: > 
+    This template illustrates the creation of a VM
+    together with a NFS share
+
+parameters:
+    user_key:
+        type: string
+        label: ssh_user_key
+        description: Public user ssh key to be injected in the cluster VMs
+        constraints:
+            - custom_constraint: nova.keypair
+    user_net1:
+        type: string
+        label: user_net1
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_net2:
+        type: string
+        label: user_net2
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_share_name:
+        type: string
+        label: share_name
+        description: Give a name to the share
+        constraints:
+            - length: { min: 1, max: 32 }
+              description: Share name must contain between 1 and 32 characters
+            - allowed_pattern: "[a-zA-Z0-9_]+"
+              description: Only alpha-numeric and "_" characters allowed
+    user_share_size:
+        type: number
+        label: share_size
+        description: Specify the share size (in GB)
+        constraints:
+            - range: { min: 1, max: 100 }
+    user_vm_flavor:
+        type: string
+        label: vm_flavor
+        description: Choose a flavor
+        constraints:
+            - custom_constraint: nova.flavor
+        default: m1.small
+    user_vm_image:
+        type: string
+        label: vm_image
+        description: Choose an image
+        constraints:
+            - custom_constraint: glance.image
+
+resources:
+    my_server:
+        type: OS::Nova::Server
+        depends_on:
+            - my_share
+        properties:
+            name: My_VM_with_NFS_share
+            metadata:
+                # Some metadata examples to be included in the VM
+                server: master
+                color: red
+                share_export_loc_path: { get_attr: [my_share, export_locations, 0, path] } 
+            security_groups: 
+                - default
+            networks: 
+                - network: { get_param: user_net1 }
+                  tag: master_ip
+                - network: { get_param: user_net2 }
+                  tag: master_ip
+            key_name: { get_param: user_key }
+            image: { get_param: user_vm_image }
+            flavor: { get_param: user_vm_flavor }
+            user_data: |
+                #!/usr/bin/env bash
+                apt-get -y install jq
```

Comment:
This wont work with a centos/redhat image. Maybe we should detect the OS first and use yum/apt or just avoid to use jq to parse the DATA output..

Reply:
I would use python for this, instead of an external app to parse json. That should work from any OS, the json package is already there...

Reply:
I'll try in Python.